web/core/src/jsMain/kotlin/org/jetbrains/compose/web/events/SyntheticEvent.kt

```diff
@@ -0,0 +1,26 @@
+package androidx.compose.web.events
+
+import org.w3c.dom.HTMLElement
+import org.w3c.dom.HTMLInputElement
+import org.w3c.dom.events.Event
+import org.w3c.dom.events.EventTarget
+
+open class SyntheticEvent<Element : HTMLElement>(
+    val target: Element,
+    val nativeEvent: Event
+) {
+    val bubbles: Boolean = nativeEvent.bubbles
+    val cancelable: Boolean = nativeEvent.cancelable
+    val composed: Boolean = nativeEvent.composed
+    val currentTarget: HTMLElement? = nativeEvent.currentTarget.unsafeCast<HTMLInputElement?>()
+    val eventPhase: Short = nativeEvent.eventPhase
+    val defaultPrevented: Boolean = nativeEvent.defaultPrevented
+    val timestamp: Number = nativeEvent.timeStamp
```

Comment:
I think it should be Int

Reply:
in kotlin/js nativeEvent the type is Number, so I followed its example.

Did you want to suggest that it's better to cast to Int?

Reply:
It's bad typing, timestamp is never Double